sqlite_utils/db.py

```diff
@@ -1292,7 +1331,7 @@ def count_where(
         :param where_args: Parameters to use with that fragment - an iterable for ``id > ?``
           parameters, or a dictionary for ``id > :id``
         """
-        sql = "select count(*) from [{}]".format(self.name)
+        sql = "select count(*) from {}".format(escaped_name(self.name))
```

Comment:
you'll see this pattern lots of places. Basically, for attached tables, you cannot use the `[table_name]` escape syntax if the table name is actually `attached_db.table1` - it has to be `attached_db.[table1]`.